.github/actions/create-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,224 @@
+
+inputs:
+  token:
+    description: 'A GitHub token used for octokit and GH CLI with the required permissions'
+    required: true
+  base:
+    description: 'The branch the pull request will be merged into'
+    required: true
+  head:
+    description: 'The branch that contains the changes you want to merge'
+    required: true
+  shell-script:
+    description: 'The location of the `prepare_release.sh` shell script that creates the changelogs and version bumps'
+    required: true
+  slack-webhook:
+    description: 'The Slack webhook to use for notifications'
+    required: true
+  version-locked:
+    description: 'Whether or not the version should be locked to axe-core'
+    default: 'false'
+    required: false
+  docs-repo:
+    description: 'The name of the repo where the release notes live'
+    default: 'null'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
+
+    - name: Is release in progress
+      uses: dequelabs/axe-api-team-public/.github/actions/is-release-in-progress-v1@main
+      id: is_release_in_progress
+      with:
+        github-token: ${{ inputs.token }}
+
+    - name: Generate commit list
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }}
+      uses: dequelabs/axe-api-team-public/.github/actions/generate-commit-list-v1@main
+      id: generate_commit_list
+      with:
+        base: ${{ inputs.base }}
+        head: ${{ inputs.head }}
+
+    - name: Has releasable commits
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }}
+      uses: dequelabs/axe-api-team-public/.github/actions/has-auto-releasable-commits-v1@main
+      id: has_releasable_commits
+      with:
+        commits: '${{ steps.generate_commit_list.outputs.commit-list }}'
+        version-locked: ${{ inputs.version-locked }}
+
+    - name: Run prepare_release shell script
+      shell: bash
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }} && ${{ steps.has_releasable_commits.outputs.should_release }}
+      id: prepare_release
+      run: |
+        if [ "${{ inputs.version-locked }}" = "true" ]
+        then
+          # We will always release as patch if the version is locked to axe-core
+          bash ${{ inputs.shell-script }} patch
+        else 
+          bash ${{ inputs.shell-script }}
+        fi
+
+        # Get the version number from the prepare_release shell script and set it as an output
+
+        # Monorepos have a lerna.json file that contains the updated version number
+        # Non-monorepos, or non-npm repos, have a package.json file that contains the updated version number
+        if [ -f "lerna.json" ]; then
+          echo "version=$(cat lerna.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        else
+          echo "version=$(cat package.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        fi
```

Comment:
To keep consistent with getting the release date/notes, this should probably be it's own step rather than joined with the prepare release script